yarn.lock

```diff
@@ -3339,7 +3339,7 @@ __metadata:
     eslint-plugin-import: ^2.29.1
     eslint-plugin-react: ^7.33.2
     eslint-plugin-security: ^1.7.1
-    express: ^4.18.2
```

Comment:
why there are yarn lock changes.

Reply:
In accessibility insights Main repo this version is upgraded to 4.19.2. The branch typescript-part5 was not upto date with main thus this change is showing.

Reply:
For package.json and yarn lock taken latest changes from typescript-part5 branch and pasted as it is in these files. So these changes are not due to fixing errors